src/components/MessagePlainTextBody.vue

```diff
@@ -1,11 +1,16 @@
 <template>
 	<div>
-		<div id="mail-content" v-html="htmlBody" />
-		<div v-if="signature" class="mail-signature" v-html="htmlSignature" />
+		<div id="mail-content" v-html="nl2br(body)" />
+		<details v-if="signature" class="mail-signature">
+			<summary>{{ signatureSummaryAndBody.summary }}</summary>
+			<span v-html="nl2br(signatureSummaryAndBody.body)" />
```

Comment:
Is the body sanitized somewhere?

Reply:
yes, at https://github.com/nextcloud/mail/blob/3802480378f6160d1029169afa3334665bd76699/lib/Model/IMAPMessage.php#L436